app/javascript/angular/code/filtersUtils.js

```diff
@@ -105,3 +105,7 @@ export const clearLocation = (elmAction, params) => {
   elmAction.send(null);
   params.update({ data: { location: '' }});
 }
+
+export const toggleIndoor = (isIndoor, params) => {
+  params.update({ data: { indoor: isIndoor }});
+}
```

Comment:
Does it have to be in filters utils even if it's only used in one place?

Reply:
I guess not